TimeTable/Categories/WorkTimes/WorkTimesViewModel.swift

```diff
@@ -156,6 +154,24 @@ class WorkTimesViewModel: WorkTimesViewModelType {
         return dailyWorkTimesArray[indexPath.section]
     }
     
+    private func createTask(for indexPath: IndexPath) -> Task? {
+        guard let dailyWorkTime = dailyWorkTime(for: indexPath) else { return nil }
+        guard let workTime = workTime(for: indexPath) else { return nil }
+        let url: URL?
+        if let taskUrlString = workTime.task {
+            url = URL(string: taskUrlString)
+        } else {
+            url = nil
+        }
```

Comment:
You don't have to set `URL` to nil. It is already nil

Reply:
It is not initialized then. url is constant value